aspnetcore/security/cors/6.0sample/Cors/WebAPI/Program.cs

```diff
@@ -0,0 +1,497 @@
+#define TESTE // FIRST SECOND THIRD ENDP ATTR DC DCORS AA SA AAH ERH CCO WHX AAH AAH2 PFX
+// PFX TEST TESTE
+#if FIRST
+#region snippet
+var  MyAllowSpecificOrigins = "_myAllowSpecificOrigins";
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddCors(options =>
```

Comment:
@halter73 I moved all the code from startup to progam.cs and everything works. Is there anything to worry about migrating to the minimal hosting model and using CORS. Tested and it all works.